.github/workflows/deploy_ghpages.yml

```
@@ -1,16 +1,32 @@
 name: Deploy GitHub pages
 
 on:
+  release:
+    types: [created]
   pull_request:
   push:
     branches: master
 
+env:
+  DOC_REPO: alphacsc/alphacsc.github.io
+  ALPHACSC_BRANCH: master
     
 jobs:
+
   build_docs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - name: Cache documentation
+        if: ${{ github.ref_type }} != 'tag'
+        uses: actions/cache@v3
+        env:
+          cache-name: cache-documentation
+        with:
+          path: doc/_build/
```

I think you mainly need to cache the example no?
```suggestion
          path: |
            doc/_build/
            doc/auto_examples
```

---

Yes, actually I was thinking that sphinx used _build/doctree directory, probably your change will work then. Thanks!

Shall I update the [documentation repo](https://github.com/alphacsc/alphacsc.github.io) then?

---

If this is working, feel free to change the doc and merge :)

---

Unfortunately cache still does not work :(